plugins/lookup/onepassword.py

```diff
@@ -50,6 +50,10 @@
         version_added: 7.1.0
       vault:
         description: Vault containing the item to retrieve (case-insensitive). If absent will search all vaults.
+      major_version:
+        description: The op CLI major version. If absent will attempt to auto-detect the version.
+        type: str
+        version_added: 7.4.0
```

Comment:
I think right now only the strings `1` and `2` are allowed, right? In that case you should better add
```suggestion
        version_added: 7.4.0
        choices:
          - "1"
          - "2"
```
(same for the option for the other plugin).